src/main/java/gomedic/model/person/UniqueAbstractPersonList.java

```diff
@@ -145,7 +145,12 @@ public void setPersons(List<? extends T> persons) {
      * Returns the backing list as an unmodifiable {@code ObservableList}.
      */
     public ObservableList<T> asUnmodifiableSortedByIdObservableList() {
-        return internalUnmodifiableList;
+        return internalUnmodifiableList
+            .sorted((abstractPerson, otherAbstractPerson) ->
+                abstractPerson.getId() == otherAbstractPerson.getId()
+                    ? 0
+                    : abstractPerson.getId().getIdNumber()
+                    < otherAbstractPerson.getId().getIdNumber() ? -1 : 1);
```

Comment:
Sorry, I accidentally missed this out. Thanks! 😅

Reply:
No worries!